scapy/layers/ipsec.py

```diff
@@ -65,14 +65,21 @@ class AH(Packet):
 
     name = 'AH'
 
+    def __get_icv_len(self):
```

Comment:
Thank you for your effort, I do test the codes, it works fine. Just there might be a small inconvenience in icv data format. More information please refer comments of 
https://github.com/secdev/scapy/issues/533